app-modules/form/src/Http/Controllers/FormWidgetController.php

```diff
@@ -254,4 +270,79 @@ public function store(
             'message' => 'Form submitted successfully.',
         ]);
     }
+
+    public function registerProspect(Request $request, Form $form): JsonResponse
+    {
+        $this->validate($request, [
+            'email' => 'email|required|unique:prospects,email',
+            'first_name' => 'required|max:255',
+            'last_name' => 'required|max:255',
+            'preferred' => 'required|max:255',
+            'mobile' => 'required|max:255',
+            'bitrhdate' => 'required|date',
+            'address' => 'required|max:255',
+            'address_2' => 'required|max:255',
+            'city' => 'required|max:255',
+            'state' => 'required|max:255',
+            'postal' => 'required|max:255',
+        ]);
```

Comment:
Please refactor this into a form request class, and use array syntax for rules instead of a string with `|`.